third_party/protobuf/k8s.io/api/core/v1/generated.proto

```diff
@@ -455,6 +455,14 @@ message ConfigMap {
   // +optional
   optional k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
 
+  // Immutable, if set to true, ensures that data stored in the ConfigMap cannot
```

Comment:
as part of the rebase this got updated automatically. As mentioned in the README its not technically required we update it, but I figured since it was already done it was better to have it updated than revert it